NickvisionMoney.GNOME/Views/AccountView.cs

```diff
@@ -526,6 +526,10 @@ public AccountView(AccountViewController controller, MainWindow parentWindow, Ad
         var actAccountSettings = Gio.SimpleAction.New("accountSettings", null);
         actAccountSettings.OnActivate += AccountSettings;
         actionMap.AddAction(actAccountSettings);
+        //Toggle Side Pane Action
+        var actToggleSidePane = Gio.SimpleAction.New("toggleSidePane", null);
+        actToggleSidePane.OnActivate += (sender, e) => _flap.SetRevealFlap(!_flap.RevealFlap);
```

Comment:
Could you make this GetRevealFlap() instead of RevealFlap. I know it's better C# practice to use properties for these things...but gir.core properties don't work 100% yet, so we are using the method versions of them for now :)